bin/phpbench.bat

```diff
@@ -0,0 +1,2 @@
+@echo off
+php.exe "%~dp0\phpbench.php" %*
```

Comment:
Sorry, the above is wrong. This doesn't work `C:\Work\phpbench>bin\phpbench`.
Too many open projects /-:
It works because composer creates a bat file when installing it as a library. https://getcomposer.org/doc/articles/vendor-binaries.md#what-about-windows-and-bat-files-

However, when I run `composer update` it doesn't seem to create that file.